base9/Settings.cpp

```diff
@@ -521,11 +518,10 @@ template<typename T> T Settings::getOrRequest (Node & n, string && f)
     {
         string input = "";
         cout << "Please enter your desired setting for field: '" + f + "':\n";
-        getline(cin, input);
-        cout << "Field '" + f + "' set to: " << input << endl;
```

Comment:
Did you mean to remove this `cout`? If it was for testing, that's fine. Otherwise, perhaps it should be added back to output the value of `t` to ensure the string parse went as planned?